webapp/app/plugins/controller.js

```diff
@@ -270,7 +270,7 @@ treeherder.controller('PluginCtrl', [
                             return thBuildApi.retriggerJob($scope.repoName, requestId);
                         }
                     }).then(function() {
-                        thNotify.send("Retriggered job (" + $scope.job.job_type_symbol + ")",
+                        thNotify.send("Retriggered job (" + $scope.jobSearchStr + ")",
```

Comment:
Agreed, it looks strange with the new(?) (job_type) in paren being appended to the job name in .jobSearchStr. I will remove the outer paren. I may also try "Retriggered job: " ie. with a colon, to separate the descriptor from the job name.